### ISSUE TITLE
Перестали разрываться тела начиная с версии 2.1.417

### ISSUE THREAD

**Issue by Erken22** [OGSR/OGSR-Engine#491]
Начиная с версии 2.1.417 перестали разрываться тела когда их затягивает и закручивает вихрь, к примеру в начале на кордоне рядом с мусоркой аномалий в скриптовой сцене где мутант бежит к аномалий и его должно разорвать его не разрывает и он просто умирает.

**Comment by PRXPHET** [comment]
А он при этом крутится внутри аномалии или его выкидывает оттуда?

**Comment by muvbed** [comment]
Тоже заметил. Похоже, что не разрывает все тела, у которых есть story_id.

**Comment by PRXPHET** [comment]
> Тоже заметил. Похоже, что не разрывает все тела, у которых есть story_id.

Да, тут конфликт фиксов. Некоторые тела нельзя разрывать, а другие - это та же плоть в карусели.
